diff --git a/node_modules/react-virtualized/dist/commonjs/utils/animationFrame.js b/node_modules/react-virtualized/dist/commonjs/utils/animationFrame.js
index ca55add..392bf15 100644
--- a/node_modules/react-virtualized/dist/commonjs/utils/animationFrame.js
+++ b/node_modules/react-virtualized/dist/commonjs/utils/animationFrame.js
@@ -22,8 +22,10 @@ if (typeof window !== 'undefined') {
 var request = win.requestAnimationFrame || win.webkitRequestAnimationFrame || win.mozRequestAnimationFrame || win.oRequestAnimationFrame || win.msRequestAnimationFrame || function (callback /*: Callback*/) /*: RequestAnimationFrame*/{
   return (win /*: any*/).setTimeout(callback, 1000 / 60);
 };
+request = request.bind(win)
 var cancel = win.cancelAnimationFrame || win.webkitCancelAnimationFrame || win.mozCancelAnimationFrame || win.oCancelAnimationFrame || win.msCancelAnimationFrame || function (id /*: number*/) {
   (win /*: any*/).clearTimeout(id);
 };
+cancel = cancel.bind(win)
 var raf /*: RequestAnimationFrame*/ = exports.raf = (request /*: any*/);
 var caf /*: CancelAnimationFrame*/ = exports.caf = (cancel /*: any*/);
\ No newline at end of file
diff --git a/node_modules/react-virtualized/dist/commonjs/vendor/detectElementResize.js b/node_modules/react-virtualized/dist/commonjs/vendor/detectElementResize.js
index 29403d9..b268d62 100644
--- a/node_modules/react-virtualized/dist/commonjs/vendor/detectElementResize.js
+++ b/node_modules/react-virtualized/dist/commonjs/vendor/detectElementResize.js
@@ -35,12 +35,14 @@ function createDetectElementResize(nonce, hostWindow) {
       var raf = _window.requestAnimationFrame || _window.mozRequestAnimationFrame || _window.webkitRequestAnimationFrame || function (fn) {
         return _window.setTimeout(fn, 20);
       };
+      raf = raf.bind(_window);
       return function (fn) {
         return raf(fn);
       };
     }();
     var cancelFrame = function () {
       var cancel = _window.cancelAnimationFrame || _window.mozCancelAnimationFrame || _window.webkitCancelAnimationFrame || _window.clearTimeout;
+      cancel = cancel.bind(_window);
       return function (id) {
         return cancel(id);
       };
diff --git a/node_modules/react-virtualized/dist/es/utils/animationFrame.js b/node_modules/react-virtualized/dist/es/utils/animationFrame.js
index fc03817..d8d2a0b 100644
--- a/node_modules/react-virtualized/dist/es/utils/animationFrame.js
+++ b/node_modules/react-virtualized/dist/es/utils/animationFrame.js
@@ -16,8 +16,10 @@ if (typeof window !== 'undefined') {
 var request = win.requestAnimationFrame || win.webkitRequestAnimationFrame || win.mozRequestAnimationFrame || win.oRequestAnimationFrame || win.msRequestAnimationFrame || function (callback /*: Callback*/) /*: RequestAnimationFrame*/{
   return (win /*: any*/).setTimeout(callback, 1000 / 60);
 };
+request = request.bind(win)
 var cancel = win.cancelAnimationFrame || win.webkitCancelAnimationFrame || win.mozCancelAnimationFrame || win.oCancelAnimationFrame || win.msCancelAnimationFrame || function (id /*: number*/) {
   (win /*: any*/).clearTimeout(id);
 };
+cancel = cancel.bind(win)
 export var raf /*: RequestAnimationFrame*/ = (request /*: any*/);
 export var caf /*: CancelAnimationFrame*/ = (cancel /*: any*/);
\ No newline at end of file
diff --git a/node_modules/react-virtualized/dist/es/vendor/detectElementResize.js b/node_modules/react-virtualized/dist/es/vendor/detectElementResize.js
index 16fda9a..f613493 100644
--- a/node_modules/react-virtualized/dist/es/vendor/detectElementResize.js
+++ b/node_modules/react-virtualized/dist/es/vendor/detectElementResize.js
@@ -29,12 +29,14 @@ export default function createDetectElementResize(nonce, hostWindow) {
       var raf = _window.requestAnimationFrame || _window.mozRequestAnimationFrame || _window.webkitRequestAnimationFrame || function (fn) {
         return _window.setTimeout(fn, 20);
       };
+      raf = raf.bind(_window);
       return function (fn) {
         return raf(fn);
       };
     }();
     var cancelFrame = function () {
       var cancel = _window.cancelAnimationFrame || _window.mozCancelAnimationFrame || _window.webkitCancelAnimationFrame || _window.clearTimeout;
+      cancel = cancel.bind(_window);
       return function (id) {
         return cancel(id);
       };
